python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt_tests/dlt_tests/conftest.py

```diff
@@ -0,0 +1,26 @@
+import os
+import uuid
+from typing import Generator
+
+import dlt
+import pytest
+from dlt import Pipeline
+
+
+@pytest.fixture(autouse=True, scope="function")
+def dlt_pipeline() -> Generator[Pipeline, None, None]:
```

Comment:
big nit, but to differentiate the fixture function and the output of the fixture itself:
```suggestion
@pytest.fixture(name="dlt_pipeline", autouse=True, scope="function")
def dlt_pipeline_fixture() -> Generator[Pipeline, None, None]:
```